src/lm.jl

```diff
@@ -65,13 +65,16 @@ function deviance(r::LmResp)
     v
 end
 
-function nulldeviance(r::LmResp)
+function nulldeviance(r::LmResp, hasintercept::Bool=true)
```

Comment:
Better define this method only for `LinearModel` if it cannot be computed only from information contained in the `LmResp` object.

Reply:
Sure - there will be only one definition of nulldeviance (and as well as null loglikelihood).